hack/pre-release.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+set -euo pipefail
+# this script updates the WMCO and OCP versions in the Makefile, CSV, bundle and Dockerfiles.
+
+# Include the common.sh 
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/common.sh
+
+CVP=false
+BASEIMAGE=false
+
+# Locate and store the WMCO version from the makefile 
+# finds "WMCO_VERSION ?=" and selects text following it
```

Comment:
You’d want to move this to where you moved the variable setting

Reply:
FTR, we should hold up PRs for something that was overlooked and easily fixed in the current PR itself. Opening a follow up a PR to fix moving of comment is unnecessary overhead for everyone.